src/epub/text/act-1.xhtml

```diff
@@ -105,7 +105,7 @@
 									<br/>
 									<span>This day should Clarence closely be mew’d up,</span>
 									<br/>
-									<span>About a prophecy, which says that <i epub:type="z3998:grapheme">g</i></span>
+									<span>About a prophecy, which says that <abbr epub:type="z3998:personal-name">G</abbr></span>
```

Comment:
This is the only change that has to be made. The rest are correctly marked up as graphemes, they refer to a letter as a letter and not as anything else.